### PR TITLE
#161966964 Setup react project 

### PR DESCRIPTION
**Pull request by gwako94**
#### What does this PR do?
Sets up empty react project

#### Description of Task to be completed?
Have an empty react project set up

#### How should this be manually tested?
```
$ git clone https://github.com/andela/ah-jumanji-frontend.git
$ cd ah-jumanji-frontend
$ git checkout ch-setup-project-161966964
$ npm install
$ npm start
```
#### Checklist:
- [x] My code follows the style guidelines of this project
- [x] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## SET UP
```
Comment (mashafrancis) [inline README.md:4]: You can add the project title "Authors Haven" on the Readme as well.

**Review by mashafrancis — CHANGES_REQUESTED** [review]

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  },
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true,
+    "jasmine": true,
+    "jquery": true
+  },
+  "rules": {
+    "quotes": 0,
```
Comment (brayovsky) [inline .eslintrc.json:27]: I think we should have consistent string quotes(single quotes) use. A `quotes: ["warn", "single", { "allowTemplateLiterals": true, "avoidEscape": true }]`.

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    "new-cap": 0,
```
Comment (brayovsky) [inline .eslintrc.json]: It's best practice to leave this on.

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    "no-console": 1,
+    "no-debugger": 1,
+    "no-param-reassign": 0,
```
Comment (brayovsky) [inline .eslintrc.json:31]: It's also best practice to have constructors begin with a capital letter

File: .eslintrc.json [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    "semi": [
+      1,
+      "always"
+    ],
+    "no-trailing-spaces": 0,
+    "eol-last": 0,
+    "no-unused-vars": 0,
+    "no-underscore-dangle": 0,
+    "no-alert": 0,
+    "no-lone-blocks": 0,
```
Comment (brayovsky) [inline .eslintrc.json]: Line 37 - 42 also should be on as best practice

**Review by brayovsky — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 2 above)
Comment (gwako94) [reply .eslintrc.json:27]: @brayovsky we are referencing the Andela engineering playbook style guides.
Kindly have a look at it here: https://gist.github.com/sa-adebayo/fe846cfb1c8c16660a7343aa94d21d65

**Review by ghost — COMMENTED** [review]
I'm wondering why we are using `Express` here, a `Node.js` library. Is it necessary? It looks to me that we are adding an unnecessary dependency to the project. I stand corrected though if you can demonstrate why we need it.

File: .eslintrc.json (hunk 5 above)
Comment (gwako94) [reply .eslintrc.json]: > Line 37 - 42 also should be on as best practice

Done

File: .eslintrc.json (hunk 3 above)
Comment (gwako94) [reply .eslintrc.json]: > It's best practice to leave this on.

Done

File: .eslintrc.json (hunk 2 above)
Comment (gwako94) [reply .eslintrc.json:27]: > I think we should have consistent string quotes(single quotes) use. A `quotes: ["warn", "single", { "allowTemplateLiterals": true, "avoidEscape": true }]`.

Based on the deliberation we've had among the team members, we've resolved to have both `single` and `double` quotes in that different implementations would require either of the two, e.g `strings` and `json data`.

**Review by mashafrancis — APPROVED** [review]

**Review by GransonO — APPROVED** [review]
The PR works as expected.
The CI issues will be resolved once the testing has been set up
`LGTM`

**Review by brayovsky — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**